### PR TITLE
feat: Add curator name to a collection

### PR DESCRIPTION
**Pull request by Bento007**
### Reviewers
**Functional:** @seve

**Readability:** @ebezzi 

---

## Changes
- add database migration to add curator to collection info
- add curator name to the response of GET Collection
- add curator name from the id token to the PUT request when modifying a collection.

## Definition of Done (from ticket)
- The curator name is returned

## QA steps (optional)
https://add-curator-name-frontend.rdev.single-cell.czi.technology/
1. using the portal website create a collection. You can see `curator_name` is added to the payload when creating the collection
1. Navigate to the collection page. You will see `curator_name` in the GET response for the collection
1. Now modify the collection details. You can see `curator_name` is added to the payload when creating the collection.
1. Publish the collection, and then navigate to the collection page. You will see `curator_name` in the GET response for the collection.

### CODE REVIEW

**Review by seve — APPROVED** [review]
LGTM!

**Comment by seve** [comment]
Looks like we accidentally pushed commits directly onto the feature branch instead of writing up PR's to merge onto it. @ebezzi you can ignore any frontend changes in this PR

**Review by seve — APPROVED** [review]

File: backend/database/versions/24_32f819760df4_add_curator.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from backend.corpora.common.corpora_orm import StrippedString
+
+revision = "24_32f819760df4"
+down_revision = "23_449eea8f5aeb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```
Comment (ebezzi) [inline backend/database/versions/24_32f819760df4_add_curator.py:22]: Nit: these comments can probably go

**Review by ebezzi — APPROVED** [review]
Looks good so far, but don't we need some more code to bind the new table?

File: backend/database/versions/24_32f819760df4_add_curator.py (hunk 1 above)
Comment (Bento007) [reply backend/database/versions/24_32f819760df4_add_curator.py:22]: > Looks good so far, but don't we need some more code to bind the new table?

By bind do you mean add curator_name to `Collection` in corpora_orm.py.

**Comment by Bento007** [comment]
> Looks good so far, but don't we need some more code to bind the new table?

By bind do you mean add curator_name to `Collection` in corpora_orm.py.

**Comment by seve** [comment]
@Bento007 @ebezzi There's no new table only a new column on collection correct?

**Comment by ebezzi** [comment]
Sorry, I completely missed it - this is good to go!

**Pull request merged**